tests/e2e/models/Navigation.js

```diff
@@ -29,3 +32,14 @@ class DesktopNav {
 
   }
 }
+
+class MobileNav{
+  constructor(mobileMenu, topToolbar){
+    this.hamburgerButton = topToolbar.find('.BRmobileHamburger');
+    this.menuSearchButton = mobileMenu.find('[data-target="#mm-4"]');
+    this.searchBox = mobileMenu.find('.BRbooksearch.mobile');
+    this.searchResult = mobileMenu.find('.BRmobileSearchResult');
```

Comment:
This will match all search results, not just the first.
```suggestion
    this.searchResults = mobileMenu.find('.BRmobileSearchResult');
```